src/Storages/StorageReplicatedMergeTree.cpp

```diff
@@ -6203,7 +6205,8 @@ void StorageReplicatedMergeTree::alter(
         size_t mutation_path_idx = std::numeric_limits<size_t>::max();
 
         String new_metadata_str = future_metadata_in_zk.toString();
-        ops.emplace_back(zkutil::makeSetRequest(fs::path(zookeeper_path) / "metadata", new_metadata_str, current_metadata->getMetadataVersion()));
```

Comment:
Do we use `current_metadata->getMetadataVersion()` in other places? Should we fix the metadata when loading the table?

Reply:
done